### PR TITLE
Updated the reset bindings and started on the new GUI

### PR DESCRIPTION
**Pull request by Smeltie**
Uploaded an image for the new gui, new images following soon!
But this resetting update is crucial enough that i've decided to make this PR to push it up to main.

### CODE REVIEW

**Comment by kitlith** [comment]
in theory, in the latest commits to the feeder app, the actions should show up as "Reset" and "Fast Reset" instead of "request_calibration". That can just be a quick PR to docs once that release actually comes out though.

Otherwise, the reset binding documentation looks good to me?

**Comment by calliePepper** [comment]
Would like to see a double check on that wording before letting it through!

**Comment by Smeltie** [comment]
> Would like to see a double check on that wording before letting it through!

On it, had a busy week(end)

**Comment by Smeltie** [comment]
@calliePepper @kitlith made some changes please feel free to check them and give any feedback!
I've been a little busy, so my responses are slower than usual.

**Review by kitlith — COMMENTED** [review]
wording regarding resets mostly looks good to me. I'm going to get a new release of the feeder app out as soon as soon as i confirm that feeder is actually sending reset/fast reset actions to the server correctly.

File: server-setup/setting-reset-bindings.md [hunk 1]
```diff
@@ -17,6 +17,16 @@ In this guide we'll show you how to set them up.
 * TOC
 {:toc}
 
+# What is a reset?
+
+A reset is the action of resetting the slimeVR skeleton model to a default pose.
+This is needed to mitigate any drift you may experience overtime.
+TYou have the option of reset or fast reset, whichever you use depends on your situation.
+Reset will do a full reset where by you have to stand straight, look forward and reset (with standard 6 point tracking it is not required to t-pose).
+A fast reset is used in scenarios where you are for example laying down, in this case you can lay down straight and use the fast reset to roughly straighten your body again.
```
[the remark this answers is not in the document]
Comment (calliePepper) [reply server-setup/setting-reset-bindings.md:26]: There should be a description of why you would use each of these resets, agreed

File: server-setup/setting-reset-bindings.md [hunk 2]
```diff
@@ -17,6 +17,16 @@ In this guide we'll show you how to set them up.
 * TOC
 {:toc}
 
+# What is a reset?
+
+A reset is the action of resetting the slimeVR skeleton model to a default pose.
+This is needed to mitigate any drift you may experience overtime.
+TYou have the option of reset or fast reset, whichever you use depends on your situation.
```
[the remark this answers is not in the document]
Comment (Smeltie) [reply server-setup/setting-reset-bindings.md:24]: fixed that

File: server-setup/setting-reset-bindings.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Smeltie) [reply server-setup/setting-reset-bindings.md:26]: added that

**Comment by Smeltie** [comment]
@calliePepper @kitlith added both suggested points.

**Review by calliePepper — APPROVED** [review]

**Pull request merged**